test/server/lib/HostedStorageManager.ts

```diff
@@ -273,6 +273,17 @@ class TestStore {
     private _externalStorageCreate: (purpose: 'doc'|'meta', extraPrefix: string) => ExternalStorage|undefined) {
   }
 
+  public async run<T>(fn: () => Promise<T>): Promise<T> {
+    await this.begin();
+    let result;
+    try {
+      result = await fn();
+    } finally {
+      await this.end();
+    }
+    return result;
+  }
```

Comment:
This change is a proposal for better clarity. It's in a separate commit and it's not mandatory, if you don't like I can revert it.

In order to hide the indentation changes, you can enable the [`Hide whitespace changes` feature](https://github.blog/changelog/2021-10-14-hiding-whitespace-is-now-remembered-for-each-pull-request/) or add `?w=1` in the URL ([direct link](https://github.com/gristlabs/grist-core/pull/767/files?w=1))